### ISSUE TITLE
concat and apply transfo not working on 2d images

### ISSUE THREAD

**Issue by HarrisNami** [neuropoly/spinalcordtoolbox#1419]
### Prerequisites

* [x] Did you check past issues for similar issue(s) ?
* [x] Can you reproduce the issue with any of the files found in `sct_testing_data` ?
* [x] If you are using a development environment, is your environment updated ?

### Description
Even on 2D images the script uses 
`isct_ComposeMultiTransform 3`

### Steps to Reproduce

1. data here:
Django/folder_shared/sct_issue/20170725_harris
2. `sct_concat_transfo -d warp_sym_510to515.nii.gz -w warp_sym_510to515.nii.gz,warp_sym_515to520.nii.gz -o warp_510to520.nii.gz`

**Expected behavior:**

output is 3.4e+38 everywhere

**Actual behavior:**

What actually happens

**Reproduces how often:**

What percentage of the time does it reproduce?

### State of spinalcordtoolbox

Include the first 3 lines printed when running the command.
Example:

```
$ sct_qc

--
Spinal Cord Toolbox (github-templates/a63f51caa065655ad1b926901adf7ef144d2cc2b)
Running /Users/geper/devel/spinalcordtoolbox/scripts/sct_qc.py

...
```

### Additional Information

Screen-shots, error logs and output files.

**Comment by HarrisNami** [comment]
Same for sct_apply_transfo: `isct_antsApplyTransforms -d 3 -`

**Comment by HarrisNami** [comment]
`isct_ComposeMultiTransform 3 warp_785to800.nii.gz -R ref2D.nii.gz warp_sym_805to800.nii.gz warp_sym_810to805.nii.gz warp_sym_815to810.nii.gz warp_sym_820to815.nii.gz`

**Comment by HarrisNami** [comment]
`sct_concat_transfo -d ref2D.nii.gz -w warp_sym_820to815.nii.gz,warp_sym_815to810.nii.gz,warp_sym_810to805.nii.gz,warp_sym_805to800.nii.gz -o warp_785to800.nii.gz`

**Comment by SaraDupont** [comment]
`isct_antsApplyTransforms` is a binary so I don't have access to the code to fix it, but the bug on `sct_concat_transfo` is fixed by PR #1420
